src/test/java/ca/corefacility/bioinformatics/irida/ria/unit/web/analysis/CartControllerTest.java

```diff
@@ -191,20 +192,20 @@ public void testGetCartMap() {
 		Map<Project, Set<Sample>> selected = new HashMap<>();
 		selected.put(project, samples);
 		controller.addSelected(selected, Locale.ENGLISH);
-
-		Map<String, Object> cartMap = controller.getCartMap();
-		assertTrue(cartMap.containsKey("projects"));
-		@SuppressWarnings("unchecked")
-		List<Map<String, Object>> pList = (List<Map<String, Object>>) cartMap.get("projects");
-		Map<String, Object> projectMap = pList.iterator().next();
-
-		assertTrue(projectMap.containsKey("samples"));
-		@SuppressWarnings("unchecked")
-		List<Map<String, Object>> sList = (List<Map<String, Object>>) projectMap.get("samples");
-		for (Map<String, Object> map : sList) {
-			assertTrue(map.containsKey("id"));
-			assertTrue(map.containsKey("label"));
-		}
+//
```

Comment:
Will this code be re-added in another merge, or will it be deleted?

Reply:
Yes, this is part of that request that is going to take a while that I will update in a future merge request.